packages/grafana-data/src/valueFormats/categories.ts

```diff
@@ -45,6 +45,7 @@ export const getCategories = (): ValueFormatCategory[] => [
         id: 'short',
         fn: scaledUnits(1000, ['', ' K', ' Mil', ' Bil', ' Tri', ' Quadr', ' Quint', ' Sext', ' Sept']),
       },
+      { name: 'SI prefix', id: 'siprefix', fn: SIPrefix('') },
```

Comment:
I think this would be identical to the "short" unit

Reply:
No, it would not. For example, `10000000000` would become `10 Bil` with the `short` unit, while the proposed SI prefix unit would return `10 G`. Additionally, I don't think there's support for shortening values with exponent below 0. For example, `0.00001` would become the same with the short unit, while `SI prefix` would return `10 u`. 

Generally, I believe the SI prefix notation would be much more understandable in the engineering community, and perhaps also elsewhere. Especially since the convention for the number of zeros in billion, trillion etc varies across the globe.

Reply:
Would it not be a suffix if it's after the value?

Reply:
That's an interesting point, but I think the [common term is prefix](https://en.wikipedia.org/wiki/Metric_prefix) since the letter normally prefixes a unit. It's just that the unit in non-existent in this case. Maybe the name could be clarified to "Unitless SI prefix" or something?

Reply:
Or maybe `SI short` to present it as an alternative to the `short` notation?

Reply:
@raymalt yes, that is a better name

Reply:
@torkelo ok, I pushed a new commit with an update naming.